src/modules/chat/index.js

```diff
@@ -12,7 +12,7 @@ const channelEmotesTip = require('../channel_emotes_tip');
 const legacySubscribers = require('../legacy_subscribers');
 
 const EMOTE_STRIP_SYMBOLS_REGEX = /(^[~!@#$%\^&\*\(\)]+|[~!@#$%\^&\*\(\)]+$)/g;
-const MENTION_REGEX = /@\b(.*)\b/;
+const MENTION_REGEX = /^@([a-zA-Z\d_]+)$/;
```

Comment:
Err, are we doing @ mention for displaynames or just usernames? How does that even work? I mean, clicking on it doesn't work right? Do you just care about the slight bolding?